vimiv/version.py

```diff
@@ -36,14 +36,19 @@ def info() -> str:
         if git_info is not None
         else f"vimiv v{vimiv.__version__}"
     )
+    metadata_plugins = metadata.get_registrations()
+    metadata_plugins_version = (
+        "False"
+        if not metadata_plugins
+        else "".join(f"\n    {n}: {v if v else '-'}" for n, v in metadata_plugins)
```

Comment:
When would version ever be falsy? I would just assume that this is defined as a string

Reply:
When the plugin does not require any backend. Thich is for example the case for the internal metadata plugin. But maybe we should just ignore this special cases :thinking:

Reply:
It will still have to override the two abstract methods, no? So this one could e.g. return "internal" as name, and "N/A" as version

Reply:
No longer relevant, as *Leave as-is for now, extend :version with plugin information in a separate attempt. I can tackle this after we have this merged*